src/components/ae-input/ae-input.vue

```diff
@@ -95,18 +105,18 @@ export default {
   }
 
   &.focus {
-    border-left: 2px solid $color-focus;
-    caret-color: $color-focus;
+    border-left: 2px solid $color-magenta;
+    caret-color: $color-magenta;
   }
   &.focus .ae-input-label {
-    color: $color-focus;
+    color: $color-magenta;
   }
   &.error {
-    border-left: 2px solid $color-error;
-    caret-color: $color-error;
+    border-left: 2px solid $color-red;
+    caret-color: $color-red;
   }
   &.error .ae-input-label {
-    color: $color-error;
+    color: $color-red;
```

Comment:
I think `$color-error` has better semantic than `$color-red` for "error" state of the input.

Reply:
@davidyuk yep, you're right! I kinda went and used search replace for this, and didn't notice the class.